lib/riak/multi.rb

```diff
@@ -37,10 +37,6 @@ def initialize(client, keys)
       raise ArgumentError, t('array_type', :array => keys.inspect) unless keys.is_a? Array
 
       self.thread_count = client.multi_threads
-      unless thread_count.is_a?(Integer) && thread_count > 0
-        raise ArgumentError, t("invalid_multiget_thread_count") # TODO: should be invalid_multi_thread_count
-      end
```

Comment:
Sorry if it's obvious, why was this removed? I'm multitasking about 10 different things today.